bluebottle/recurring_donations/templates/recurring_donations/mails/monthly_donation.mail.html

```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 {% blocktrans %}
-Hi {{ receiver_first_name }},<br/>
+Hi {{ first_name }},<br/>
```

Comment:
@gannetson , if you're sure this doesn't change the translation string then I can merge this PR